index.html

```diff
@@ -301,20 +301,29 @@ <h2>Contact me</h2>
             feature that you need built or a project that needs coding. I’d love
             to help with it!
           </p>
-          <form action="#">
-            <input type="text" placeholder="full name" class="contact-input" />
+          <form action="https://formspree.io/f/{form_id}" method="post">
             <input
               type="text"
-              placeholder="@example.gmail.com"
+              maxlength="30"
+              placeholder="full name"
+              class="contact-input"
+              required
+            />
+            <input
+              type="email"
+              placeholder="example@gmail.com"
               class="contact-input"
+              required
             />
             <textarea
-              name=""
-              id=""
+              name="textarea"
+              id="textarea"
               cols="0"
               rows="6"
               placeholder="your message here..."
               class="contact-input"
+              maxlength="500"
+              required
             ></textarea>
             <input type="button" class="btn" value="Get in touch" />
```

Comment:
- [ ] Right now, this button is not submitting so kindly consider adding `submit` in the type attribute instead of `button`.
- [ ] Also the alignment of button in your design is not similar to the Figma design. kindly consider adjusting it taking help of the figma design
Figma design
![as](https://user-images.githubusercontent.com/96683843/228314878-5cede7b7-5ae8-4889-896f-69beec4d2c65.jpg)